From 3ff1971e28327c89e72f1df2d952339aed48ebeb Mon Sep 17 00:00:00 2001
From: Ronan Abhamon <ronan.abhamon@vates.fr>
Date: Thu, 15 Dec 2022 14:36:04 +0100
Subject: [PATCH 100/177] feat(fork-log-daemon): ignore SIGTERM

Without this patch, the output logs of the fork-log-daemon child
are never displayed when SIGTERM is sent to the PGID.

Signed-off-by: Ronan Abhamon <ronan.abhamon@vates.fr>
---
 scripts/fork-log-daemon | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/fork-log-daemon b/scripts/fork-log-daemon
index eb0f0b0f..665a60ba 100755
--- a/scripts/fork-log-daemon
+++ b/scripts/fork-log-daemon
@@ -1,12 +1,14 @@
 #!/usr/bin/env python
 
 import select
+import signal
 import subprocess
 import sys
 import syslog
 
 def main():
     process = subprocess.Popen(sys.argv[1:], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    signal.signal(signal.SIGTERM, signal.SIG_IGN)
     write_to_stdout = True
 
     while process.poll() is None:
